src/NetMQ/NetMQFrame.cs

```diff
@@ -337,5 +337,14 @@ public byte[] ToByteArray(bool copy = false)
 
             return byteArray;
         }
+
+        /// <summary>
+        /// Return an readonly span of bytes that carries the content of this NetMQFrames Buffer.
+        /// </summary>
+        /// <returns>the Buffer as a readonly span</returns>
+        public ReadOnlySpan<byte> ToReadOnlySpan()
+        {
+            return new ReadOnlySpan<byte>(Buffer);
```

Comment:
This should take the message length into account. The buffer could be longer than the message.